### DISCUSSION TITLE
docs/ninjutsu/special-up-ranking-ninja/single-line-reader

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#10] [General]
# docs/ninjutsu/special-up-ranking-ninja/single-line-reader

官网地址：极简单行阅读器

https://996.ninja/docs/ninjutsu/special-up-ranking-ninja/single-line-reader

**Comment by BBFTZ** [comment]
6

**Comment by 18674815526** [comment]
6翻了

**Comment by Andrewjonesss** [comment]
有MAC吗

**Reply by MingGH** [reply]
mac 没得，可以用 xbar 整一个

**Reply by Xav1erSue** [reply]
我正在开发一个跨平台的 [https://github.com/cloak-app/cloak](https://github.com/cloak-app/cloak)
支持 macOS/Windows/Linux，目前还在开发中，大火如果有需要的可以先试试看，帮忙提提需求和BUG

**Comment by Mr-Second** [comment]
需要手机号？
